apis/sql/postgresql/v1alpha1/grant_types.go

```diff
@@ -20,13 +20,19 @@ import (
 	"context"
```

Comment:
This file with all its contents will be injected into provider sql and as I understand more GRANTS will be issued is that right?

Reply:
This is copy pasted direclty from the provider-sql. We import it to be able to create valid CRs for the provider.

Reply:
This is them being added :D